components/kyma-environment-broker/internal/auditlog/templates/extra_conf_kyma.yaml

```diff
@@ -0,0 +1,34 @@
+[INPUT]
```

Comment:
This is not a `yaml` file, so we can drop the .yaml suffix. Instead, we can use something like `.conf`.

Reply:
done